README.md

```diff
@@ -1,23 +1,50 @@
 
-Welcome to Node Feature Discovery Operator!
+## How it Works
 
-### See our [Documentation][documentation] for detailed instructions and reference
+The operator works by orchestrating all resources needed to run the Node-Feature-Discovery DaemonSet. This DaemonSet runs on each node in the cluster and detects the features and configurations of the node's hardware. The operator then collects and organizes this information, making it easily accessible.
 
-## Community, discussion, contribution, and support
+## Quick start
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+Get the source code
 
-You can reach the maintainers of this project at:
+```bash
+git clone -b v0.5.0 https://github.com/kubernetes-sigs/node-feature-discovery-operator
+```
 
-- [Slack channel](https://kubernetes.slack.com/messages/node-feature-discovery)
-- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-node)
+Deploy the operator
 
-### Code of conduct
+> By default it will deploy using the minimal tag image, is
+> desired you can simply modify the IMAGE_TAG env var to point to the image
+> tag to use.
 
-Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
+```bash
+IMAGE_TAG={{ site.container_image }}
+make deploy
+```
 
-[documentation]: https://kubernetes-sigs.github.io/node-feature-discovery-operator
\ No newline at end of file
+By default the operator will watch `NodeFeatureDiscovery` objects
+only in the namespace where the operator is deployed in. This is
+specified by the `WATCH_NAMESPACE` env variable in the operator
+deployment manifest. If unset the operator will watch ALL
+namespaces.
+
+Create a NodeFeatureDiscovery instance
+
+```bash
+kubectl apply -f config/samples/nfd.kubernetes.io_v1_nodefeaturediscovery.yaml
+```
+
+## Documentation
+
+For more detailed information on how to use the Node Feature Discovery operator, please check out our [documentation](https://github.com/kubernetes-sigs/node-feature-discovery-operator/blob/master/docs/index.md)
+
+## Contributing
+
+The Node Feature Discovery operator welcomes contributions, and interested parties are encouraged to take a look at the [contributing guidelines](CONTRIBUTING.md) and [open issues](https://github.com/kubernetes-sigs/node-feature-discovery-operator/issues). We're excited to have you join our community of contributors.
+
+## Support
+
+If there are any issues or questions about the Node Feature Discovery operator, they can be addressed by opening an issue on the [GitHub repository](https://github.com/kubernetes-sigs/node-feature-discovery-operator/issues) or reaching out on the [Slack channel](https://kubernetes.slack.com/messages/node-feature-discovery-operator).
```

Comment:
You could point to https://github.com/kubernetes-sigs/node-feature-discovery-operator/issues/new/choose for new issues